front/pages/poke/[wId]/index.tsx

```diff
@@ -329,9 +328,215 @@ const WorkspacePage = ({
                   </DropdownMenu.Items>
                 </DropdownMenu>
               </div>
+              <div className="border-material-200 my-4 flex flex-col rounded-lg border p-4">
```

Comment:
😱 Can you move this to a dedicated table like we did for other tables?

Reply:
Yes @flvndvd 😄